torch/nn/parallel/distributed.py

```diff
@@ -625,24 +637,26 @@ class DistributedDataParallel(Module, Joinable):
     # used to track whether the given thread is inside ddp forward for torchdynamo purposes
     _active_ddp_module: Optional["DistributedDataParallel"] = None
 
+    reducer: dist.Reducer
+
     def __init__(
         self,
-        module,
-        device_ids=None,
-        output_device=None,
-        dim=0,
-        broadcast_buffers=True,
-        process_group=None,
-        bucket_cap_mb=None,
-        find_unused_parameters=False,
-        check_reduction=False,
-        gradient_as_bucket_view=False,
-        static_graph=False,
+        module: Module,
+        device_ids: Optional[Sequence[Union[int, device]]] = None,
+        output_device: Optional[Union[int, device]] = None,
+        dim: int = 0,
+        broadcast_buffers: bool = True,
+        process_group: "Optional[ProcessGroup]" = None,
```

Comment:
I changed this type, so now it is different than in the `pyi` file. From the code it seems it should be like this. Though I had to add a `# type: ignore[assignment]` to line 679.